forest/benchmarking/tests/conftest.py

```diff
@@ -56,10 +57,10 @@ def cxn():
 @pytest.fixture(scope='session')
 def benchmarker():
     try:
-        bm = get_benchmarker()
-        bm.apply_clifford_to_pauli(Program(I(0)), sI(0))
```

Comment:
This was probably related to rigetti/pyquil#849 in case you were wondering.

Reply:
It definitely is related! :)